sdk/identity/azcertificatemanager/autorest.md

```diff
@@ -0,0 +1,33 @@
+# File with heading
+
+## Go
+
+``` yaml
+
+clear-output-folder: false
+export-clients: true
+go: true
+input-file: https://github.com/Azure/azure-rest-api-specs-pr/blob/f9a0ac622533987324a0a18237af12ce4f526563/specification/pki/data-plane/Microsoft.Pki/preview/2022-09-01-preview/pki.json
+license-header: MICROSOFT_MIT_NO_VERSION
+module: github.com/Azure/azure-sdk-for-go/sdk/identity/azcertificatemanager
+openapi-type: "data-plane"
+output-folder: ../azcertificatemanager
+use: "@autorest/go"
```

Comment:
You should pin to a specific version.  The latest at the time of this writing is `@autorest/go@4.0.0-preview.45`